docs/src/fs_event.rst

```diff
@@ -8,6 +8,16 @@ FS Event handles allow the user to monitor a given path for changes, for example
 if the file was renamed or there was a generic change in it. This handle uses
 the best backend for the job on each platform.
 
+.. note::
+    For AIX, the non default IBM bos.ahafs package has to be installed.
```

Comment:
can we link to some documentation here? or I guess AIX people are usually familiar with how to do that?